lib/i18n/tasks/deepl_translation.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'deepl'
+require 'i18n/tasks/html_keys'
+
+module I18n::Tasks
+  module DeeplTranslation
+    # @param [I18n::Tasks::Tree::Siblings] forest to translate to the locales of its root nodes
+    # @param [String] from locale
+    # @return [I18n::Tasks::Tree::Siblings] translated forest
+    def deepl_translate_forest(forest, from)
+      forest.inject empty_forest do |result, root|
+        translated = translate_list(root.key_values(root: true), to: root.key, from: from)
+        result.merge! Data::Tree::Siblings.from_flat_pairs(translated)
+      end
+    end
+
+    # @param [Array<[String, Object]>] list of key-value pairs
+    # @return [Array<[String, Object]>] translated list
+    def translate_list(list, opts) # rubocop:disable Metrics/AbcSize
```

Comment:
Consider prefixing all methods here and in `google_translation.rb` with `google_` / `deepl_` and extracting common methods into a `base_translation.rb` module.

Reply:
I prefixed the methods with `google_` / `deepl_`. The only method is `*_to_values(list)` that would identify for a `base_translation.rb` Do you think it's worth sharing?

Reply:
Not worth it unless we re-structure it to share more methods. I can look into it after merging, thanks.